package.json

```diff
@@ -21,16 +21,16 @@
     "start-emulator-persist": "gcloud beta emulators datastore start --host-port=:15606 --use-firestore-in-datastore-mode",
     "start-emulator": "gcloud beta emulators datastore start --host-port=:15606 --no-store-on-disk --use-firestore-in-datastore-mode",
     "stop-emulator": "curl -X POST 'http://localhost:15606/shutdown'; pkill -f \"CloudDatastore.jar\"",
-    "build": "gulp",
-    "watch": "gulp watch",
+    "build": "tsc && gulp",
+    "watch": "concurrently \"tsc --watch\" \"gulp watch\"",
```

Comment:
This didn't work for me, but it doesn't seem to work on the main branch either, and I don't use it, but others might want to.

Reply:
could you clarify what did not work? it seems fine when I run it

Reply:
oh i see. it a concurrency issue. sometimes gulp gets running before tsc compiles